demisto_sdk/__main__.py

```diff
@@ -672,6 +697,71 @@ def id_set_command(**kwargs):
     id_set_creator.create_id_set()
 
 
+# ====================== update-release-notes =================== #
+@main.command(name="update-release-notes",
+              short_help='''Auto-increment pack version and generate release notes template.''')
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    "-p", "--pack", help="Name of the pack."
+)
+@click.option(
+    '-u', '--update_type', help="The type of update being done. [major, minor, revision]",
+    type=RNInputValidation()
+)
+@click.option(
+    '--all', help="Update all changed packs", is_flag=True
+)
+@click.option(
+    "--pre_release", help="Indicates that this change should be designated a pre-release version.",
+    is_flag=True)
+def update_pack_releasenotes(**kwargs):
+    _pack = kwargs.get('pack')
+    update_type = kwargs.get('update_type')
+    pre_release = kwargs.get('pre_release')
+    is_all = kwargs.get('all')
+    modified, added, old, _packs = FilesValidator(use_git=True).get_modified_and_added_files()
+    packs_existing_rn = set()
+    for pf in added:
+        if 'ReleaseNotes' in pf:
+            pack_with_existing_rn = get_pack_name(pf)
+            packs_existing_rn.add(pack_with_existing_rn)
+    if len(packs_existing_rn):
+        existing_rns = ''.join(f"{p}, " for p in packs_existing_rn)
+        print_warning(f"Found existing release notes for the following packs: {existing_rns.rstrip(', ')}")
+    if len(_packs) > 1:
+        pack_list = ''.join(f"{p}, " for p in _packs)
+        if not is_all:
+            print_error(f"Detected changes in the following packs: {pack_list.rstrip(', ')}")
+            if _pack:
+                pass
+            else:
+                sys.exit(0)
+    if len(modified) < 1:
+        print_warning('No changes were detected.')
+        sys.exit(0)
+    if is_all and not _pack:
+        packs = list(_packs - packs_existing_rn)
+        packs_list = ''.join(f"{p}, " for p in packs)
+        print_warning(f"Adding release notes to the following packs: {packs_list.rstrip(', ')}")
+        for pack in packs:
+            update_pack_rn = UpdateRN(pack=pack, update_type=update_type, pack_files=modified,
+                                      pre_release=pre_release)
+            update_pack_rn.execute_update()
+    elif is_all and _pack:
+        print_error(f"Please remove the --all flag when specifying only one pack.")
+        sys.exit(0)
+    else:
+        if _pack:
+            if _pack in packs_existing_rn:
+                print_error(f"Release notes are already updated for {_pack}")
```

Comment:
lets write something like that:
f"New release notes file already found for {_pack}. please update manually or delete {specific_pack_new_rn_file}"